### PR TITLE
#324 Add missing parts for web components docs

### PR DESCRIPTION
**Pull request by lojzatran**

### CODE REVIEW

File: extension/docs/WebComponentsIntegrationGuide.md [hunk 1]
```diff
@@ -513,6 +516,26 @@ A CTP payment with `submitAdditionalPaymentDetailsResponse` field with the respo
...
+  "actions": [
+    {
+      "action": "setCustomField",
+      "name": "submitAdditionalPaymentDetailsRequest",
+      "value": "{\"details\":{\"threeds2.challengeResult\":\"eyJ0cmFuc1...\"}}"
+    },
+    {
+      "action": "setCustomField",
+      "name": "submitAdditionalPaymentDetailsResponse",
+      "value": ""
```
Comment (ahmetoz) [inline extension/docs/WebComponentsIntegrationGuide.md:532]: actually it's better to use describe it without value (unset)

Comment (ahmetoz) [inline extension/docs/WebComponentsIntegrationGuide.md:532]: is it better to describe it without value ? (unset)

Comment (lojzatran) [reply extension/docs/WebComponentsIntegrationGuide.md]: Ah correct: https://github.com/commercetools/commercetools-adyen-integration/pull/337/commits/c64bd199642694466b3d96f31fe07f3b4fa21c29

**Comment by lojzatran** [comment]
Close because this branch has some weird conflicts.

**Pull request closed without merging**